save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/UploadVulnerabilityView.kt

```diff
@@ -45,7 +45,7 @@ val uploadVulnerabilityView: VFC = VFC {
                             className = ClassName("needs-validation")
 
                             fileManagerComponent {
-                                // todo: storage size shouldn't be more then 500MB per project
+                                // FixMe: need to add the correct directory for COSV files
```

Comment:
not directory -- we have a dedicated API to upload COSV

Reply:
Fixed it